lib/headlines/security_headers/x_frame_options.rb

```diff
@@ -15,7 +15,7 @@ def score_by_value
       end
 
       def score_by_allow_from
-        whitelisted_domain? ? 1 : 0
+        whitelisted_domain? ? 1 : -1
```

Comment:
@codelion 
According to the X-Frame-Options section under HTTP tab in the Scoring Matrix (https://docs.google.com/spreadsheets/d/11egIRbFc2865cFuz5X-o51GmOePQDnexcZY50NYnofU/edit#gid=0), we don't have a score of zero at all. 
So we realized that we had a wrong score initially. This line fixes it.